UnitTests/MultipleTransformations.cs

```diff
@@ -0,0 +1,74 @@
+using NUnit.Framework;
+
+namespace JUST.UnitTests
+{
+    [TestFixture]
+    public class MultipleTransformations
+    {
+        [Test]
+        public void MultipleTransformsScalarResult()
+        {
+            const string input = "{\"d\": [ \"one\", \"two\", \"three\" ], \"values\": [ \"z\", \"c\", \"n\" ]}";
+            const string transformer = 
+                "{ \"result\": " +
+                    "{ \"#transform($)\": [ " + 
+                        "{ \"condition\": { \"#loop($.values)\": { \"test\": \"#ifcondition(#stringcontains(#valueof($.d[0]),#currentvalue()),True,yes,no)\" } } }, " +
+                        "{ \"intermediate_transform\": \"#valueof($.condition)\" }," +
+                          "\"#exists($.intermediate_transform[?(@.test=='yes')])\" ] } }";
```

Comment:
Please consider using raw string literals (e.g. "triple quotes") to make these JSONs more readable.
See: https://devblogs.microsoft.com/dotnet/csharp-11-preview-updates/

(Of course, you'll need to bump the language version to 11.)